packs/csi_openstack_cinder/templates/csi_openstack_cinder.nomad.tpl

```diff
@@ -0,0 +1,84 @@
+job [[ template "job_name" . ]] {
+  [[ template "region" . ]]
+
+  datacenters = [[ .csi_openstack_cinder.datacenters | toPrettyJson ]]
+
+  type = system
+
+  group "nodes" {
+
+    restart {
+      attempts = [[ .csi_openstack_cinder.job_restart_config.attempts ]]
+      delay    = [[ .csi_openstack_cinder.job_restart_config.delay ]]
+      mode     = [[ .csi_openstack_cinder.job_restart_config.mode ]]
+      interval = [[ .csi_openstack_cinder.job_restart_config.interval ]]
+    }
```

Comment:
I get the following error when running this pack:

```
$ nomad-pack run -var=cloud_conf_file=$(pwd)/cloud.conf .
! Failed To Parse Job Specification

        Error:   400 Bad Request
        Type:    client.GenericOpenAPIError
        Context:
                 - Template Name: csi_openstack_cinder/templates/csi_openstack_cinder.nomad.tpl
                 - Registry Name: dev
                 - Pack Name: csi_openstack_cinder
                 - Pack Ref: dev
                 - Deployment Name: csi_openstack_cinder
```

I then used `nomad-pack render -var=cloud_conf_file=$(pwd)/cloud.conf .` and copied the results to a file (see https://github.com/hashicorp/nomad-pack/issues/163 for the bug of not being able to redirect that to a file), and I get the following:

```
$ nomad job validate ./cinder.nomad
Error getting job struct: Error parsing job file from ./cinder.nomad:
cinder.nomad:14,20-21: Missing newline after argument; An argument definition must end with a newline.
```

The jobspec is:

<details><summary>cinder.nomad</summary>

```hcl
job "csi_openstack_cinder" {
  region = "global"

  datacenters = [
  "dc1"
]

  type = system

  group "nodes" {

    restart {
      attempts = 5
      delay    = 15s
      mode     = delay
      interval = 5m
    }


  constraint {
    attribute = "${attr.platform.aws.placement.availability-zone}"
    value     = "nova"
  }task "cinder-node" {
      driver = "docker"
      template {
        data        = <<EOT
[Global]
auth-url="https://keystone.cloud.example.net:5000/v3"
application-credential-id="f9f4aaf8-3680-11ec-89d6-0242ac110002"
application-credential-secret="imAvwSfkjuzw2ocaieF9t9+vAGRJ3YpBdnQ/+IUeoa0="
        EOT
        destination = "secrets/cloud.conf"
        change_mode = "restart"
      }
      config {
        image = "docker.io/k8scloudprovider/cinder-csi-plugin:latest"

        mount {
            type     = "bind"
            target   = "/etc/config/cloud.conf"
            source   = "./secrets/cloud.conf"
            readonly = false
            bind_options {
              propagation = "rshared"
            }
          }
        args = [
  "/bin/cinder-csi-plugin",
  "-v=3",
  "--endpoint=unix:///csi/csi.sock",
  "--cloud-config=/etc/config/cloud.conf"
]
        privileged = true
      }

      csi_plugin {
        id        = "csi-cinder"
        type      = "node"
        mount_dir = "/csi"
      }
    }
    task "cinder-controller" {
      driver = "docker"
      template {
        data        = <<EOT
[Global]
auth-url="https://keystone.cloud.example.net:5000/v3"
application-credential-id="f9f4aaf8-3680-11ec-89d6-0242ac110002"
application-credential-secret="imAvwSfkjuzw2ocaieF9t9+vAGRJ3YpBdnQ/+IUeoa0="
        EOT
        destination = "secrets/cloud.conf"
        change_mode = "restart"
      }
      config {
        image = "docker.io/k8scloudprovider/cinder-csi-plugin:latest"
        mount {
            type     = "bind"
            target   = "/etc/config/cloud.conf"
            source   = "./secrets/cloud.conf"
            readonly = false
            bind_options {
              propagation = "rshared"
            }
          }

        args = [
  "/bin/cinder-csi-plugin",
  "-v=3",
  "--endpoint=unix:///csi/csi.sock",
  "--cloud-config=/etc/config/cloud.conf"
]

      }

      csi_plugin {
        id        = "csi-cinder"
        type      = "controller"
        mount_dir = "/csi"
      }
    }
  }
}
```

</details>

It looks like the `delay`, `mode`, and `interval` are all strings and need to be quoted:

```suggestion
    restart {
      attempts = [[ .csi_openstack_cinder.job_restart_config.attempts ]]
      delay    = "[[ .csi_openstack_cinder.job_restart_config.delay ]]"
      mode     = "[[ .csi_openstack_cinder.job_restart_config.mode ]]"
      interval = "[[ .csi_openstack_cinder.job_restart_config.interval ]]"
    }
```

Reply:
Got these fixed up.